parameters/src/canary/resources/set_validator_state.metadata

```diff
@@ -0,0 +1,6 @@
+{
+  "prover_checksum": "e8fbb86008343485bb95c3e3e75d41b493ba654cc161d8b9f9c6901cb71d7a9d",
+  "prover_size": 17390188,
+  "verifier_checksum": "d0942ee642048e21cbf562737b4760524dd378ac17867e7a225736badb23f614",
+  "verifier_size": 673
```

Comment:
The parameter scripts should optimally be re-run for both Testnet and Mainnet as well.

Reply:
It works fine as is